### PR TITLE
a ton of mobile improvements to save space

### LINKED ISSUE #4334: /booking: save more horizontal space by making columns smaller
before
CleanShot 2022-09-09 at 13 54 23@2x

### LINKED ISSUE #4337: fix app bar  with frosted glass, make CTA fixed on mobile
https://github.com/calcom/cal.com/pull/4336

### PR DESCRIPTION
**Pull request by PeerRich**
- reduced column size in /booking
- made AppBar and Bottom Nav fixed with frosted glass on mobile
- hiding headline and description on mobile
- fixed CTA on bottom right on mobile

/bookings before:

CleanShot 2022-09-09 at 14 45 57@2x

/bookings after:
CleanShot 2022-09-09 at 14 46 10@2x

demo:
https://user-images.githubusercontent.com/8019099/189352382-3b890ca2-89e3-46b7-8f2f-fbe38d89132c.mp4

recorded on iPhone 12 Mini:

https://user-images.githubusercontent.com/8019099/189358338-18d6654c-4842-4d78-8c4e-a2f9a9198ed7.mp4

### CODE REVIEW

**Review by sean-brydon — APPROVED** [review]
Super clean :)

File: apps/web/styles/globals.css [hunk 1]
```diff
@@ -220,6 +220,10 @@ button[role="switch"][data-state="checked"] {
   .bottom-nav {
     padding-bottom: 24px;
   }
+
+  .cta{
```
Comment (emrysal) [inline apps/web/styles/globals.css:224]:
```suggestion
  .cta {
```

**Review by emrysal — DISMISSED** [review]
Unit test fail

**Review by emrysal — DISMISSED** [review]
Definitely a unit test issue, the clarity could be improved though.

```
TypeError: window.getEmbedTheme is not a function

      372 | if (isBrowser) {
      373 |   const url = new URL(document.URL);
    > 374 |   embedStore.theme = (window.getEmbedTheme() || "auto") as UiConfig["theme"];
```

**Comment by PeerRich** [comment]
thank you for merging!

**Pull request merged**